src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,6 +124,12 @@ private int RunExecutable()
                 return result;
             }
 
+            // Add Nuget Packages Probing Path
+            var nugetPackagesRoot = _context.PackagesDirectory;
```

Comment:
Can you create a method AddNuGetPackagesProbingPath and encapsulate these four lines of code and call it here. Then remove the comment. Much cleaner.

Also, no tests?